cli/cmd/compose/run.go

```diff
@@ -29,17 +29,18 @@ import (
 )
 
 type runOptions struct {
-	Name        string
-	Command     []string
-	WorkingDir  string
-	ConfigPaths []string
-	Environment []string
-	Detach      bool
-	Remove      bool
+	Name    string
+	Command []string
+	Detach  bool
```

Comment:
Note that we have `Name` and `Detach` in the `composeOptions` struct so we're shadowing them here. We should remove these fields as we did `ConfigPaths`, etc.